reconcile/openshift_base.py

```diff
@@ -433,17 +433,23 @@ def validate_data(oc_map, actions):
                 ready_replicas = status.get('readyReplicas')
                 if not desired_replicas == replicas == \
                         ready_replicas == updated_replicas:
-                    logging.info('new replicas not ready, status is invalid')
+                    logging.info(
+                        f'{kind} {name} has replicas that are not ready '
+                        f'({ready_replicas} ready / {desired_replicas} total)')
                     raise ValidationError(name)
             elif kind == 'Subscription':
                 state = status.get('state')
                 if state != 'AtLatestKnown':
-                    logging.info('Subscription status.state is invalid')
+                    logging.info(
+                        f'Subscription {name} state is invalid. '
+                        f'Current state: {state}')
                     raise ValidationError(name)
             elif kind == 'Job':
                 succeeded = status.get('succeeded')
+                conditions = status.get('conditions')
```

Comment:
you can move this line inside the if below

Reply:
btw, can this return `None`? If yes, maybe the default should be and empty string? (to avoid the message `Job conditions are: None`). Not sure if it's any better.

Reply:
Thanks for pointing that out.

I moved retrieving the conditions inside of the if and since the value is actually either `None` or a list of dict, instead of changing to default to empty string I checked if there are conditions (not None or empty list) and dump them as yaml for better readability instead of printing them raw

Reply:
sounds great, tks!